mypy/stubgen.py

```diff
@@ -1713,12 +1716,17 @@ def generate_stubs(options: Options) -> None:
         with generate_guarded(mod.module, target, options.ignore_errors, options.verbose):
             generate_stub_for_c_module(mod.module, target, sig_generators=sig_generators)
     num_modules = len(py_modules) + len(c_modules)
-    if not options.quiet and num_modules > 0:
+    if not options.quiet and (num_modules > 0 or len(ignored_py_modules) > 0):
         print("Processed %d modules" % num_modules)
         if len(files) == 1:
             print(f"Generated {files[0]}")
         else:
             print(f"Generated files under {common_dir_prefix(files)}" + os.sep)
+        if len(ignored_py_modules):
```

Comment:
```suggestion
        if ignored_py_modules:
```

is enough.

Reply:
Thank you for reviewing!
I fixed this, and also clarify the reason why some modules are not processed by [aab77a6](https://github.com/python/mypy/pull/13919/commits/aab77a65608717a8f8fa9aa220a2929c9c78693d)